uplink/config.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink
+
+import (
+	"context"
+	"errors"
+
+	"github.com/vivint/infectious"
+	"github.com/zeebo/errs"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/metainfo/kvmetainfo"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/pointerdb/pdbclient"
+	"storj.io/storj/pkg/storage/buckets"
+	ecclient "storj.io/storj/pkg/storage/ec"
+	"storj.io/storj/pkg/storage/segments"
+	"storj.io/storj/pkg/storage/streams"
+	"storj.io/storj/pkg/storj"
+)
+
+// RSConfig is a configuration struct that keeps details about default
```

Comment:
I'm not sure whether RSConfig/EncryptionConfig, is the final correct location, but I think it should be fine for now.

Reply:
yea but I have no better idea at the moment